### PR TITLE
Prefer trakt title over plex to avoid editionTitle extra request

### LINKED ISSUE #1436: Sync fails when processing multiple editions of a film
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists
- [X] I have tried downgrading to find version that can be used as a workaround

### The problem

When syncing films it appears that Trakt sync fails when processing a film with multiple editions.

I've checked that it's not an issue with my plex or trakt sync configuration as TV shows sync successfully only movies are impacted.

### Error trace / logs

```text
adamus1red $:/docker/plex $ docker compose run --rm plextraktsync --batch-delay 1 --server Plextra sync --sync movies
INFO     PlexTraktSync [0.25.16-16-g8da8ddc_main]
INFO     Sync Movie sections: ['Films']
WARNING  Partial walk, disabling liked lists updating. Liked lists won't update because it needs full library sync.
Processing Films  18% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╺━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 248/1,414  [ 0:00:34 < -:--:-- , ? it/s ]
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py", line 449, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py", line 444, in _make_request
    httplib_response = conn.getresponse()
                       ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/http/client.py", line 1374, in getresponse
    response.begin()
  File "/usr/local/lib/python3.11/http/client.py", line 318, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/http/client.py", line 279, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/socket.py", line 706, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/ssl.py", line 1278, in recv_into
    return self.read(nbytes, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/ssl.py", line 1134, in read
    return self._sslobj.read(len, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TimeoutError: The read operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/requests/adapters.py", line 489, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py", line 787, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/urllib3/util/retry.py", line 550, in increment
    raise six.reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/urllib3/packages/six.py", line 770, in reraise
    raise value
  File "/usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py", line 451, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/usr/local/lib/python3.11/site-packages/urllib3/connectionpool.py", line 340, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='10-100-10-1.c1f1f7d34a7443ea91bb07217aa206a4.plex.direct', port=32400): Read timed out. (read timeout=30)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/app/plextraktsync/__main__.py", line 18, in <module>
    cli()
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/plextraktsync/cli.py", line 25, in wrap
    cmd(*args, **kwargs)
  File "/app/plextraktsync/commands/sync.py", line 68, in sync
    runner.sync(walker=w, dry_run=config.dry_run)
  File "/app/plextraktsync/sync.py", line 68, in sync
    self.sync_watched(movie, dry_run=dry_run)
  File "/app/plextraktsync/sync.py", line 164, in sync_watched
    logger.info(f"Marking as watched in Trakt: '{m.title}'")
                                                 ^^^^^^^
  File "/app/plextraktsync/media.py", line 47, in title
    return self.plex.title
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/functools.py", line 1001, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/app/plextraktsync/plex/PlexLibraryItem.py", line 102, in title
    if self.type == "movie" and self.item.editionTitle:
                                ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/plexapi/base.py", line 503, in __getattribute__
    self._reload(_overwriteNone=False)
  File "/usr/local/lib/python3.11/site-packages/plexapi/base.py", line 377, in _reload
    data = self._server.query(key)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/plexapi/server.py", line 745, in query
    response = method(url, headers=headers, timeout=timeout, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/requests_cache/session.py", line 103, in get
    return self.request('GET', url, params=params, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/requests_cache/session.py", line 159, in request
    return super().request(method, url, *args, headers=headers, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/requests_cache/session.py", line 206, in send
    response = self._send_and_cache(request, actions, cached_response, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/requests_cache/session.py", line 230, in _send_and_cache
    response = super().send(request, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/requests/adapters.py", line 578, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='10-100-10-1.c1f1f7d34a7443ea91bb07217aa206a4.plex.direct', port=32400): Read timed out. (read timeout=30)
```

### Expected behavior

Film is processed and updated in trakt 

### Steps to reproduce the behavior

1. Add a movie with multiple editions i.e. Lord of the Rings
2. run a sync with plex trakt sync
3. 

### Inspect of problematic items

_No response_

### Workarounds

_No response_

### Install method

docker-compose

### Config file contents

```yaml
cache:
  path: /app/config/trakt_cache
excluded-libraries:
- Private
- Family Holidays
config:
  dotenv_override: true
plex:
  timeout: 60
logging:
  append: true
  console_time: false
  debug: false
  filename: plextraktsync.log
  filter_loggers: null
  filter: null
sync:
  plex_to_trakt:
    collection: true
    clear_collected: false
    ratings: true
    watched_status: true
    watchlist: true
  trakt_to_plex:
    liked_lists: true
    ratings: true
    watched_status: true
    watchlist: true
    watchlist_as_playlist: false
watch:
  add_collection: false
  remove_collection: false
  scrobble_threshold: 80
  username_filter: true
  media_progressbar: true
xbmc-providers:
  movies: imdb
  shows: tvdb
```

### Version

0.25.16-16-g8da8ddc_main

### Python Version

3.11.2

### Operating System and Version

Linux-5.10.0-21-amd64-x86_64-with

### PR DESCRIPTION
**Pull request by twolaw**
Every output of movie title in logs or console needs an extra request to Plex server to check the `editionTitle` and display it if available.
This PR removes this extra request by displaying trakt title instead.

Trakt movie title already contains edition title (Extended, Director's cut'...) :
```
https://trakt.tv/movies/the-lord-of-the-rings-the-fellowship-of-the-ring-extended-2001
https://trakt.tv/movies/star-wars-episode-v-the-empire-strikes-back-special-edition-1997-02-21
https://trakt.tv/movies/blade-runner-director-s-cut-702424
```

Note : [Multiple Editions](https://support.plex.tv/articles/multiple-editions/) for Plex movies is a Plex Pass feature.

fixes #1436

### CODE REVIEW

File: plextraktsync/commands/inspect.py [hunk 1]
```diff
@@ -37,8 +38,13 @@ def inspect_media(id: str):
...
     print(f"Title: {media.title}")
-    if media.type == 'movie' and media.editionTitle:
-        print(f"Edition Title: {media.editionTitle}")
+    if media.type == 'movie':
+        try:
+            edition = media.editionTitle
+        except (ReadTimeout, RequestException):
+            edition = None
+        if edition:
+            print(f"Edition Title: {edition}")
```
Comment (glensc) [inline plextraktsync/commands/inspect.py:47]: add method to PlexLibraryItem and use that instead of doing this in inspect directly on pm.item

File: plextraktsync/media.py [hunk 2]
```diff
@@ -43,10 +43,9 @@ def __init__(
 
     @property
     def title(self):
-        if self.plex:
-            return self.plex.title
-
-        return f"{self.trakt.title} ({self.trakt.year})"
+        if self.trakt:
+            return f"{self.trakt.title} ({self.trakt.year})"
+        return self.plex.title
```
Comment (glensc) [inline plextraktsync/media.py:48]: move this changeset to separate pr (with explanation etc)?

Comment (twolaw) [reply plextraktsync/media.py:48]: This is the purpose of this PR.

File: plextraktsync/plex/PlexLibraryItem.py [hunk 3]
```diff
@@ -96,11 +97,21 @@ def library(self):
 
         return self.plex.library_sections[self.item.librarySectionID]
 
+    @cached_property
+    def edition_title(self):
+        if self.type == "movie":
+            try:
+                edition = self.item.editionTitle
+            except (ReadTimeout, RequestException):
```
[the remark this answers is not in the document]
Comment (twolaw) [reply plextraktsync/plex/PlexLibraryItem.py:105]: Ok, thanks to https://github.com/pkkid/python-plexapi/pull/1093 I understand the problem.
Will rework it.

**Comment by twolaw** [comment]
replaced by #1441

**Comment by glensc** [comment]
You probably want to delete your unmerged `edition-title` branch manually
